Graphics/GraphicsEngineOpenGL/src/SwapChainGLImpl.cpp

```diff
@@ -83,6 +83,13 @@ SwapChainGLImpl::SwapChainGLImpl(IReferenceCounters*       pRefCounters,
     //Set dummy width and height until resize is called by the app
     m_SwapChainDesc.Width  = 1024;
     m_SwapChainDesc.Height = 768;
+#elif PLATFORM_EMSCRIPTEN
+    //TODO: Understand why resize doesn't work
```

Comment:
Should be fixed

Reply:
This was an erroneous comment, I rechecked